accessing
pseudo
	self error: 'There is no user logged'